selene_sdk/utils/config_utils.py

```diff
@@ -132,6 +137,124 @@ def initialize_model(model_configs, train=True, lr=None):
     return model, criterion
 
 
+def create_data_source(configs, output_dir=None, load_train_val=True, load_test=True):
+    """
+    Construct data source(s) used in training/evaluation.
```

Comment:
can you please elaborate, as this comment doesn't really describe what the method is doing

Reply:
fixed in 0074139